packages/calypso-products/src/feature-group-plan-map.ts

```diff
@@ -79,8 +79,8 @@ export const featureGroups: Partial< FeatureGroupMap > = {
 			FEATURE_CONTACT_FORM_JP,
 			FEATURE_STATS_JP,
 			FEATURE_CUSTOM_DOMAIN,
-			FEATURE_LIVE_CHAT_SUPPORT,
 			FEATURE_SUPPORT_EMAIL,
+			FEATURE_LIVE_CHAT_SUPPORT,
```

Comment:
Needed to have Live Chat after Customer support to match their grouping form the design.